components/SupportFilter.js

```diff
@@ -48,13 +52,22 @@ const getNewFilters = ({ checked, name, checkedFilters }) => {
 }
 
 const SortButton = styled.div.attrs(({ sortTypeSelected }) => ({
-  className: `py-1.75 px-6 w-1/3 text-center border border-solid border-lightgray rounded-2.5 ${
+  className: `py-1.75 px-6 w-1/3 text-center border-r last:border-none border-solid border-midgray ${
```

Comment:
that's cool I didn't know you could do last: things in tailwind

Reply:
pretty good huh https://tailwindcss.com/docs/configuring-variants